bash/ez390

```diff
@@ -25,6 +25,9 @@ fi
 java -classpath ${jardir}/z390.jar -Xrs $J2SEOPTIONS ez390 $1 $2 $3 $4 $5 $6 $7 $8 $9
 
 rc=$?
+# executed program: rc = any value (R15 at exit; set in sz390)
+# ez390: rc = 1 for Java issues; 16 for GUAM GUI window closed due to timeout
+#        rc = 16 for execution errors; 4,8 when using TEST (set in sz390)
```

Comment:
Comment on RC here but not acted on.. were you going to add some output based on the result??

Reply:
No. This is just documentation. Followed what Abe did in the bat files. See xZ390.BAT (x=A,E,L,M).